### PR TITLE
Salt 2016.3.1 support for Raspberry Pi 2 and 3

### PR DESCRIPTION
**Pull request by dmurphy18**

### CODE REVIEW

File: file_roots/debian_pkg.sls [hunk 1]
```diff
@@ -3,6 +3,22 @@
 include:
 {% if buildcfg.build_release == 'debian8' %}
 
+{% if buildcfg.build_arch == 'armhf' %}
```
Comment (bbinet) [inline file_roots/debian_pkg.sls]: Why the pkg list is different for armhf build_arch ?
Shouldn't it be the same as other architectures?

Does it mean the support of armhf devices is only partial at the moment?

**Comment by dmurphy18** [comment]
@bbinet The difference is due to Raspbian including more packages than the original Debian 8 (jessie) release, hence only building those packages they are needed which are mostly for Raet and Tornado

**Comment by bbinet** [comment]
Ok, but it seems that some packages also exist in Debian 8 already, for example `python-msgpack`:
```
$ apt-cache policy python-msgpack
python-msgpack:
  Installed: (none)
  Candidate: 0.4.2-1
  Version table:
     0.4.2-1 0
        500 http://httpredir.debian.org/debian/ jessie/main amd64 Packages
```
Are there any reason those packages are still being built by salt-pack?

**Comment by dmurphy18** [comment]
@bbinet It may have been oversight, or the package was not of a sufficient level when I first built for Debian 8. A quick check informs me, it was the former.  I shall go through the list of dependencies and check the versions, using Debian 8.1 as a reference for the next point release.

Thank you for the heads up.

**Comment by bbinet** [comment]
Thanks for you efforts on salt-pack, they are very appreciated as I plan to use salt-pack to manage my own packages.

**Pull request merged**